### PR TITLE
Added coveralls.io badge to README.md

### LINKED ISSUE #83: Add coveralls.io badge to README.md
Add coveralls.io badge to README.md file which can be found on https://coveralls.io/github/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io

### PR DESCRIPTION
**Pull request by cmdrwebdev**
**Status**:
- [x] Ready
- [ ] In Development
- [ ] On Hold

**Which issue does this PR fix?**: fixes #83 

Brief description of what this PR does.
Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber"

**Why do we need this PR?**:

If relevant, please include a screenshot.

**Demo (optional)**:

Some tips for you to write the instructions:
- Prefer bulleted description
- Start after checking out this branch
- Include any setup required, such as migrating databases, etc.

**Testing instructions:**:

If there is any work still left to do, please add it here.

**TODOs (if any)**:

**A picture of a cute animal (not mandatory but encouraged)**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
  [![Build Status](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io.svg?branch=develop)](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io)
 
+[![Coverage Status](https://coveralls.io/repos/github/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/badge.svg?branch=develop)](https://coveralls.io/github/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io?branch=develop)
+
```
Comment (vaibhavsingh97) [inline README.md]: Travis CI and coveralls coverage should come in one line

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Comment by vaibhavsingh97** [comment]
@cmdrwebdev when you create PR it's should describe what this PR does and which issue this PR fixed so use `fixes #23` where 23 is the issue number. Although, you don't have to remember the issue number just write # and start writing the issue title, GitHub will automatically suggest issues. Please answer 2nd and 3rd question. You can refer [here](https://help.github.com/articles/closing-issues-using-keywords/)

File: README.md (hunk 1 above)
Comment (cmdrwebdev) [reply README.md]: Sorry @vaibhavsingh97  that was my first contribution. For the PR do i need to request another PR or just push commit and edit my existing comment to update my answer to question 2 and 3?

Comment (vaibhavsingh97) [reply README.md]: @cmdrwebdev First of all congratulations!! :tada: :tada: You have made your first PR and it needs minor changes. You have to push commit again in order to reflect new changes.

**Review by nikhita — APPROVED** [review]

**Comment by nikhita** [comment]
@cmdrwebdev Congratulations on getting your first PR merged!!! :tada: :tada: :tada:

**Comment by cmdrwebdev** [comment]
Thanks @nikhita

**Pull request merged**